src/src/main/java/org/redfrog404/mobycraft/entity/EntityChaosMonkey.java

```diff
@@ -287,13 +276,13 @@ public boolean getCanSpawnHere() {
 				return false;
 			}
 
-			return i > this.rand.nextInt(j) ? false : super.getCanSpawnHere();
+			return i <= this.rand.nextInt(j) && super.getCanSpawnHere();
```

Comment:
A little propositional logic streamlining here... Removing the ternary operator.  Should be exactly the same logic, but worth checking :-).